packages/xo-web/src/xo-app/home/index.js

```diff
@@ -1143,47 +1151,30 @@ export default class Home extends Component {
             ) : (
               <div>
                 {(type === 'VM' || type === 'host') && (
-                  <Tooltip
-                    content={
-                      selectedPowerState == null
-                        ? undefined
-                        : _(`${selectedPowerState}Item`, { type: `${type}s` })
+                  <OverlayTrigger
+                    trigger='click'
+                    rootClose
+                    placement='bottom'
+                    overlay={
+                      <Popover
+                        className={styles.selectObject}
+                        id='powerStatePopover'
+                      >
+                        <Select
+                          autoFocus
+                          multi
+                          onChange={this._updateSelectedPowerStates}
+                          openOnFocus
+                          options={
+                            type === 'VM' ? POWER_STATE_VM : POWER_STATE_HOST
+                          }
+                          value={selectedPowerStates}
+                        />
+                      </Popover>
                     }
                   >
-                    <OverlayTrigger
-                      trigger='click'
-                      rootClose
-                      placement='bottom'
-                      overlay={
-                        <Popover
-                          className={styles.selectObject}
-                          id='powerStatePopover'
-                        >
-                          <Select
-                            autoFocus
-                            onChange={this._updateSelectedPowerState}
-                            openOnFocus
-                            options={
-                              type === 'VM' ? POWER_STATE_VM : POWER_STATE
-                            }
-                            simpleValue
-                            value={selectedPowerState}
-                          />
-                        </Popover>
-                      }
-                    >
-                      <Button btnStyle='link'>
-                        <Icon
-                          icon={
-                            selectedPowerState == null
-                              ? 'unknown'
-                              : selectedPowerState
-                          }
-                        />{' '}
```

Comment:
Why did you remove it? Don't you want to keep at least some sort of icon?

Reply:
I delete it because we added the multi possibility and I don't like that the icon is displayed (one state) / hidden (multiple state or nothing) according to the case
I think no icon is better.

Reply:
Yes, that makes sense but we can still keep a global icon for the selector the same way we put icons next to the other selectors. Something like [this](https://fontawesome.com/v4.7.0/icon/power-off).

Reply:
Indeed, I was not sure to use it because I had the impression that it was for "power off". 
anyway, we don't use it in XO so I can use it for the power state